src/libsync/propagateupload.cpp

```diff
@@ -217,6 +217,21 @@ void PropagateUploadFileCommon::start()
     const auto slashPosition = path.lastIndexOf('/');
     const auto parentPath = slashPosition >= 0 ? path.left(slashPosition) : QString();
 
+
+    if (!_item->_renameTarget.isEmpty() && _item->_file != _item->_renameTarget) {
+        // Try to rename the file
+        const auto originalFilePathAbsolute = propagator()->fullLocalPath(_item->_file);
+        const auto newFilePathAbsolute = propagator()->fullLocalPath(_item->_renameTarget);
+        QFile originalFile(originalFilePathAbsolute);
```

Comment:
replace `()` by `{}` that are more consistent with style of current c++
they are called universal initialization operators

Reply:
Do you think we should init every object from now on this way?